pandas/core/arrays/categorical.py

```diff
@@ -1847,6 +1847,43 @@ def __iter__(self):
         """Returns an Iterator over the values of this Categorical."""
         return iter(self.get_values().tolist())
 
+    @staticmethod
+    def _contains(key, categories, container):
+        """Returns True if `key` is in `categories` and the
+        location of `key` in `categories` is in `container`.
+
+        This is a helper method used in :method:`Categorical.__contains__`
+        and in :class:`CategoricalIndex.__contains__`.
+        """
```

Comment:
As this is not a one-liner doc-string:

* The docstring should start underneath the triple parentheses.
* The first line is a summary should only be one line
* List of parameters and their dtypes
* Return description / type